components/lib/push-amazon/build.gradle

```diff
@@ -13,6 +13,8 @@ android {
     defaultConfig {
         minSdkVersion config.minSdkVersion
         targetSdkVersion config.targetSdkVersion
+
+        buildConfigField("String", "AMAZON_DEVICE_MESSAGING_VERSION", "\"" + Versions.amazon_device_messaging + "\"")
```

Comment:
How are apps supposed to use this? They won't be able to use that in their gradle scripts - but it sounds like this was a goal here? 🤔